<?xml version="1.0" encoding="UTF-8"?><testrun name="MVCtest" project="Mjukvarutestning" tests="12" started="12" failures="1" errors="0" ignored="0">
  <testsuite name="tests.MVCtest" time="0.494">
    <testcase name="test_CheckDiceListSize" classname="tests.MVCtest" time="0.024"/>
    <testcase name="Test_RemoveDieFromList_ToBeRethrown" classname="tests.MVCtest" time="0.003"/>
    <testcase name="test_MenuChoiceExceptionTest_NotInteger" classname="tests.MVCtest" time="0.001"/>
    <testcase name="Test_RemoveDupliceFromList_AllToBeRethrown" classname="tests.MVCtest" time="0.02">
      <failure>java.lang.AssertionError: Expected exception: java.lang.IllegalArgumentException&#13;
	at org.junit.internal.runners.statements.ExpectException.evaluate(ExpectException.java:32)&#13;
	at org.junit.contrib.java.lang.system.ExpectedSystemExit$1.evaluate(ExpectedSystemExit.java:130)&#13;
	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:48)&#13;
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)&#13;
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</failure>
    </testcase>
    <testcase name="test_ThrowAllDicesResultOutput" classname="tests.MVCtest" time="0.414"/>
    <testcase name="Test_RemoveDicesFromList_3ToBeRethrown" classname="tests.MVCtest" time="0.002"/>
    <testcase name="Test_RemoveDieFromList_ToBeRethrown_OneDie_OneOutOfRange" classname="tests.MVCtest" time="0.001"/>
    <testcase name="Test_RemoveAllFromList_AllToBeRethrown" classname="tests.MVCtest" time="0.002"/>
    <testcase name="Test_SystemExit_Menu" classname="tests.MVCtest" time="0.001"/>
    <testcase name="test_ThrowAllDicesWithDifferentValues_Mock" classname="tests.MVCtest" time="0.022"/>
    <testcase name="test_MenuChoiceExceptionTest_NotInRange_TooHigh" classname="tests.MVCtest" time="0.001"/>
    <testcase name="test_MenuChoiceExceptionTest_NotInRange_TooLow" classname="tests.MVCtest" time="0.002"/>
  </testsuite>
</testrun>
